### PR TITLE
Smaller default offer

### PR DESCRIPTION
**Pull request by thomash-acinq**
Shaved 32 bytes from the default offer by removing the pathId.

### CODE REVIEW

**Comment by thomash-acinq** [comment]
We could save even more by replacing the trampoline node id with a `ShortChannelIdDir`.

File: src/commonMain/kotlin/fr/acinq/lightning/wire/OfferTypes.kt [hunk 1]
```diff
@@ -781,12 +781,12 @@ object OfferTypes {
                 nodeParams: NodeParams,
                 trampolineNode: NodeUri,
                 features: Features,
-                pathId: ByteVector32,
```
Comment (t-bast) [inline src/commonMain/kotlin/fr/acinq/lightning/wire/OfferTypes.kt:784]: nit: can you update the documentation line 776 accordingly? It should probably be:

```
@param blindingSecret session key used for the blinded path included in the offer.
```

**Review by t-bast — COMMENTED** [review]
Nice catch, this is a good quick win :+1:

File: src/commonMain/kotlin/fr/acinq/lightning/wire/OfferTypes.kt (hunk 1 above)
Comment (thomash-acinq) [reply src/commonMain/kotlin/fr/acinq/lightning/wire/OfferTypes.kt:784]: Done

File: src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt [hunk 2]
```diff
@@ -240,15 +240,14 @@ data class NodeParams(
      * This offer will stay valid after restoring the seed on a different device.
      * We also return the path_id included in this offer, which should be used to route onion messages.
```
[the remark this answers is not in the document]
Comment (thomash-acinq) [reply src/commonMain/kotlin/fr/acinq/lightning/NodeParams.kt]: Done

**Pull request merged**